e2e/user/user-can-edit-account.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect } from "@playwright/test";
+import { loginKnownUser } from "../_lib/pom/init/login-known-user";
+import { gotoUserMyAccountPage } from "../_lib/sequences/nav-gotos";
+import { UserMyAccountEditPage } from "../_lib/pom/pages/user-my-account-edit-page";
+
+test("user can login, go to edit page, but cancel, and should not see their details changed", async ({
```

Comment:
The test description should not have "but cancel" in this case.

Reply:
Addressed 76415d0b7a43453a12a8e9f100ff2ea5c5349abf